seed/views/v3/users.py

```diff
@@ -123,7 +123,8 @@ def __init__(self, *args):
                     name='New User Fields',
                     required=True,
                     description="An object containing meta data for a new user: "
-                                "required - [last_name, role(viewer/owner/member), email]",
+                                "- Required - first_name, last_name, email "
+                                "- Optional - role viewer(default), member, owner",
```

Comment:
Let's make this more clear that the field is `role`, and we're listing the valid role values next to it. Just a suggestion, so feel free to put whatever makes sense to you here.
```suggestion
                                "- Optional - role ['viewer'(default), 'member', or 'owner']",
```